### PR TITLE
Fix runtime identifier being ignored

### PR DESCRIPTION
**Pull request by LarssonOskar**
The --runtime-identifier parameter is being ignored when building containers from project. It always defaults to "linux-x64"

### CODE REVIEW

**Comment by LarssonOskar** [comment]
Bug reported in #120

File: src/Aspirate.Processors/Resources/Project/ProjectProcessor.cs [hunk 1]
```diff
@@ -124,7 +124,7 @@ public override ComposeService CreateComposeEntry(KeyValuePair<string, Resource>
             .WithEnvironment(environment)
             .WithContainerName(resource.Key)
             .WithRestartPolicy(RestartMode.UnlessStopped)
-            .WithPortMappings(ports.Select(x=> new Port
+            .WithPortMappings(ports.Select(x => new Port
```
Comment (LarssonOskar) [inline src/Aspirate.Processors/Resources/Project/ProjectProcessor.cs:127]: Formatting is on the house 😃

Comment (prom3theu5) [reply src/Aspirate.Processors/Resources/Project/ProjectProcessor.cs:127]: haha

**Comment by prom3theu5** [comment]
Looks good - thanks 😄

**Pull request merged**